README.md

```diff
@@ -22,6 +22,11 @@
 | 🛡️ Status   | [![scitools](https://img.shields.io/badge/scitools-ownership%20pending-yellow)](https://github.com/bjlittle/geovista/issues/167)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
 |              |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
 
+<p align="center">
+<img src="https://raw.githubusercontent.com/bjlittle/geovista-media/2023.09.1/media/readme/ww3-tri.png" height="250">
+<br>
+<a href="#wavewatch-iii"><b>Less than 20 lines of Python!</b></a>
+</p>
```

Comment:
I love the intention, but it seems a wee bit odd floating about here ... perhaps we could weave the `less than 20 lines of Python` in the actual example below instead?

What do you think?

Reply:
I was looking for some way of quickly showing what GeoVista was, before getting into further detail. Do you think that maybe should just be an image? Or maybe it's not worth it?

Reply:
Okay, how's about a slight rejig instead ...

Rather than have the above (which repeats the WW3 plot already in the `README.md`) we move the [Unreal Reels](https://github.com/bjlittle/geovista?tab=readme-ov-file#unreal-reels) section at the end of the `README.md` to here. 

i.e., Sell the major eye candy animation up front rather than saving it until the end, and rebrand it as something like `Bring Your Data Alive`, or something like that?

I think the succinctness of the code examples sell themselves. Let's woo the audience, get them engaged, and they may be tempted to read and explore more ...

What do you think?

Reply:
1abe1eb6d0a4ef9b8b26d6f3df401d8631186ac0

Love the idea, I'm not 100% on my execution. See what you think

Reply:
Great 💯 

Let's roll with this! We can always tweak later if we want 👍